specification/compute/resource-manager/Microsoft.Compute/preview/2021-06-01-preview/examples/GetDiskInspection.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "location": "eastus"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "x-ms-request-id": "27b7c568-16ec-46f3-bcf1-5bea3f2529b1"
+      },
+      "body": {
+        "properties": {
```

Comment:
We should not have this level. This is the keyword of the schema definition, which should not appear in the value of that schema
```suggestion
```

Reply:
Updated